docs/guide/previews.md

```diff
@@ -65,7 +71,7 @@ class ExampleComponentPreview < ViewComponent::Preview
 end
 ```
 
-You can also set a custom layout to be used by default for previews as well as the preview index pages via the `default_preview_layout` configuration option:
```

Comment:
I _believe_ this line is correct as it stands, but could be written more clearly.